PWGEM/PhotonMeson/TableProducer/createEMReducedEvent.cxx

```diff
@@ -50,15 +50,28 @@ struct createEMReducedEvent {
     kUndef = -1,
   };
 
+  // Configurable for filter/cuts
+  Configurable<int> minN_PCM{"minN_PCM", 2, "Minimum number of V0s for PCM. Events are saved if either minimum number condition is met"};
+  Configurable<int> minN_PHOS{"minN_PHOS", 2, "Minimum number of clusters for PHOS. Events are saved if either minimum number condition is met"};
+  Configurable<int> minN_EMC{"minN_EMC", 2, "Minimum number of clusters for EMCal. Events are saved if either minimum number condition is met"};
```

Comment:
I would suggest to set default value 0 to minN cut. It is different from software trigger.